src/Compilers/CSharp/Portable/Microsoft.CodeAnalysis.CSharp.csproj

```diff
@@ -7,6 +7,8 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
     <GenerateMicrosoftCodeAnalysisCommitHashAttribute>true</GenerateMicrosoftCodeAnalysisCommitHashAttribute>
+    <IsTrimmable Condition="'$(TargetFramework)' != 'netstandard2.0'">true</IsTrimmable>
+    <EnableAotAnalyzer Condition="'$(TargetFramework)' != 'netstandard2.0'">true</EnableAotAnalyzer>
```

Comment:
What is the actual target framework set where it is trimmable: `net6.0` and higher?

Reply:
Yes. I didn't remember offhand how to compare TFM versions in MSBuild safely.

Reply:
I always look here when I want to remember how to do this 

https://github.com/dotnet/roslyn/blob/main/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L6-L28

I think we should centralize this in maybe `Settings.props` and have a simpler `<RoslynEnableAot>true</RoslynEnableAot>` style approach. Dont' care about the name really.

Reply:
I don't think this works. `Settings.props` gets imported before the projects, so none of the properties from the project can impact the condition.

And `Imports.targets` gets imported after the framework targets, so properties set there can't impact the framework tooling.